src/fetchapi/http.lib.js

```diff
@@ -0,0 +1,52 @@
+const apiRequests = {
+  get: async url => {
+    let response = await fetch(url);
+    if (response.ok) {
+      const data = await response.json();
+      return data;
+    }
+  },
+  post: async (url, payload) => {
+    let response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-type': 'application/json'
+      },
+      body: JSON.stringify(payload)
+    });
+    if (response.ok) {
+      const data = await response.json();
+      return { data, response };
+    }
+  },
+  put: async (url, payload) => {
+    let response = fetch(url, {

```

Comment:
shouldn't be here `await`?

Reply:
Ye, that's for sure. How did I missed that...